### PR TITLE
Remove error crate

### PR DESCRIPTION
**Pull request by sfackler**
This is a holdover from before the standard library Error could be
downcast.

Does this seem right @reem?

This may or may not be a breaking change - not sure how others would feel.

### CODE REVIEW

**Comment by Hoverbear** [comment]
Seems reasonable for me... Definitely might be breaking though.

**Comment by untitaker** [comment]
Yeah that's a breaking change. We can merge it into master, but not any of the maintenance branches.

**Comment by untitaker** [comment]
Thanks!

File: src/error.rs [hunk 1]
```diff
@@ -1,10 +1,9 @@
-use std::error::Error as StdError;
 use std::fmt;
 
 use modifier::Modifier;
 use {Response};
 
-pub use err::Error;
+pub use std::error::Error;
```
Comment (sfackler) [inline src/error.rs]: We may want to cut this reexport. I only had it there as a (failed) attempt to keep the change roughly backwards compatible.

Comment (untitaker) [inline src/error.rs]: Ok, feel free to make a PR

**Pull request merged**